### DISCUSSION TITLE
When can support HuaWei？

### DISCUSSION THREAD

**Discussion by EricaLi123** [react-native-maps/react-native-maps#4294] [Feature requests]
### Feature request
Need the support for Huawei. Current only supports IOS and ANDROID.

### Why it is needed
Huawei doesn't have Google play.

**Comment by marcofuentes05** [comment]
Is this still being considered? It would be really useful.

**Comment by ChristianPastenC** [comment]
Is there no way to use this package on Huawei OS ?

**Comment by liu-chao** [comment]
I would like to know if this is still being considered or not? anyone replies? thanks!
